packages/paste-codemods/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@twilio-paste/codemods",
+  "version": "0.0.1",
+  "status": "beta",
+  "description": "A collection of codemods for maintaining projects built with Paste.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "bin": "./bin/paste-codemod.js",
+  "scripts": {
+    "build": "node ./tools/create-package-mappings.js",
+    "test": "jest"
+  },
+  "dependencies": {
+    "chalk": "^2.4.2",
+    "execa": "^3.2.0",
+    "globby": "^10.0.1",
+    "inquirer": "^7.0.0",
+    "is-git-clean": "^1.1.0",
+    "jscodeshift": "^0.11.0",
+    "meow": "^5.0.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.6.4",
+    "@babel/plugin-proposal-object-rest-spread": "^7.6.2",
+    "@babel/preset-env": "^7.6.3"
```

Comment:
Is there any reason these are different versions to the ones we hoist to root of the repo? In workspaces we can rely on the hoisted versions so potentially no need to declare them here if they don't need to be different to the ones we already have.

Reply:
Copy pasta, fixing. Good catch!